server/backend/api/app/crud/crud_permission_audit.py

```diff
@@ -0,0 +1,45 @@
+from sqlalchemy.orm import Session
+from sqlalchemy import and_
+from sqlalchemy import desc  # Import if you want descending order
```

Comment:
Wondering if this is unused import?

Reply:
yeah, it's unused, leftover from a stack overflow copy pasta

Reply:
I'll remove it

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1587/commits/fbecc2dd05109f516e3a81a2c66d68390757c7a5